### ISSUE TITLE
`paywallsDidFullyLoad` is not a supported event type for ApphudSdkEvents.

### ISSUE THREAD

**Issue by gevgasparyan** [apphud/ApphudSDK-React-Native#59]
When I call
```
Apphud.start({ apiKey: 'myApiKey'})
```
I'm getting this error
<img width="443" src="https://github.com/apphud/ApphudSDK-React-Native/assets/13519034/fc304b5f-36e1-428f-8e40-414c3cad9ceb" >

also it shows these two errors 
```
WARN  Sending `paywallsDidFullyLoad` with no listeners registered.
WARN  Sending `apphudDidFetchStoreKitProducts` with no listeners registered.
```

package.json
```
"dependencies": {
    "@apphud/react-native-apphud-sdk": "^1.1.0",
    "react": "18.2.0",
    "react-native": "0.71.8",
  },
```

**Comment by ryskin** [comment]
same

**Comment by gevgasparyan** [comment]
> same, I think in JS exists a method that does not exist in native code, see screenshots.
> 
> <img alt="image" width="380" src="https://user-images.githubusercontent.com/3756722/279093644-b3e83966-ca46-48ca-b37f-1915e54ae61b.png"> <img alt="image" width="608" src="https://user-images.githubusercontent.com/3756722/279093837-d0093a8e-f09c-442a-bf1c-7bc1a70e7f45.png">

Manually adding `paywallsDidFullyLoad = 'paywallsDidFullyLoad'` to the supported events fixed the issue

**Comment by ryskin** [comment]
@ren6 @alexandr-makarov-am can you guys fix this in the library?

**Comment by ren6** [comment]
@ryskin @gevgasparyan thanks guys! Fixed in 1.2.0

**Comment by gevgasparyan** [comment]
Thanks @ren6 !
